tox.ini

```diff
@@ -43,7 +43,9 @@ commands =
 passenv = HOME
 basepython = python3
 changedir = ./examples/
-deps = mechanical-markdown
+deps =
+    mechanical-markdown
+    grpcio-status>=1.60.0
 commands =
     ./validate.sh metadata
```

Comment:
You need to actually register the `error_handling` folder here as well, so we can run this via `tox -e examples`.